src/sst/elements/vanadis/inst/vinst.h

```diff
@@ -109,7 +109,7 @@ class VanadisInstruction
             count_isa_fp_reg_in(copy_me.count_isa_fp_reg_in),
             count_isa_fp_reg_out(copy_me.count_isa_fp_reg_out)
         {
-            trapError             = copy_me.trapError;
+            trap_error_           = copy_me.trap_error_;
```

Comment:
Don't want to add tails to the rest of the member variables? I think it's fine if you don't since there are lots of other formatting issues with vanadis.

Reply:
I'll update a few more but since this is a base class and protected vars, the changes cascade and a lot of files need updates. Eventually I'll fix the rest.